lib/between_meals/repo/svn.rb

```diff
@@ -91,6 +96,23 @@ def valid_ref?(ref)
       rescue
         raise Changeset::ReferenceError
       end
+
+      private
+
+      def parse_status(changes)
+        # http://svnbook.red-bean.com/en/1.0/re26.html
+        changes.lines.map do |line|
+          case line
+          when (/^([\w ])\w?\s+(\S+)$/)
```

Comment:
If the first letter is a space, you want the second letter which you won't capture here... I thnk?